spec/features/editing_content/insert_contact_embed_spec.rb

```diff
@@ -1,18 +1,27 @@
 # frozen_string_literal: true
 
-RSpec.feature "Insert contact embed", js: true do
+RSpec.feature "Insert contact embed" do
   include AccessibleAutocompleteHelper
 
-  scenario do
+  scenario "with javascript", js: true do
     given_there_is_an_edition
     when_i_go_to_edit_the_edition
     and_i_click_to_insert_a_contact
-    and_i_select_a_contact
+    and_i_select_a_contact_from_the_autcomplete
     then_i_see_the_contact_preview
     when_i_click_insert_contact
     then_i_see_the_snippet_is_inserted
   end
 
+  scenario "without javascript", js: false do
```

Comment:
Do you need to explicitly set javascript to false here for there to be no javascript?

Reply:
No - I just wanted to be explicit, rather than an implicit. Do you think it's better without?

Reply:
Hmm, what came to my mind when I saw it was 'does the js flag from the scenario prior leak into the second scenario so you need to explicitly set it in the the second'. But now I can see you're just trying to be explicit, I think it's fine!